### ISSUE TITLE
Several of the sub-components of BDropdown are not fully implemented

### ISSUE THREAD

**Issue by dwgray** [bootstrap-vue-next/bootstrap-vue-next#2244]
### Missing feature is from

bootstrap-vue

### Docs

https://bootstrap-vue.org/docs/components/dropdown#dropdown-supported-sub-components

### Describe the missing feature

Three of the helper components for `BDropdown` don't implement any of the props described in the BSV documentation. They are
- `BDropdownForm`
- `BDropdownHeader`
- `BDropdownText`

### Validations

- [X] Have tested with the latest version. This is still alpha version and sometime things change rapidly.
- [X] Follow our [Code of Conduct](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided link is to the documentation of the missing feature.

**Comment by xvaara** [comment]
since we use attr inheritance, we don't implement some props. ie. in form invalid etc. Can you check the pull request if you think something important is missing?

**Comment by dwgray** [comment]
Thanks @xvaara. I agree in general that use of attr inheritance when consistently applied will solve most of what I saw. That will require some migration docs, which I've started on, and the act of writing that will give me an idea if anything important is missing. I hope to be able to finish that off in the next day or so.

I am pretty strongly on the side of leaving the :text prop in for these and similar components. I think the DX is smoother for some use cases (including the ones that @VividLemon mentioned in #2257. In addition, removing the text props makes migration and writing the migration guide harder. I don't see that maintaining them is a heavy lift, but maybe I'm missing something.
